### ISSUE TITLE
Add internationalization steps in documentation

### ISSUE THREAD

**Issue by GledsonAfonso** [get-alex/alex#241]
### Subject of the issue

Since there's a correlation between [get-alex/alex](alex), [words/cuss](cuss) and [retextjs/retext-equality](equality), we need some help updating the [contributing.md](contributing) file adding internationalization steps.

As mentioned in #202, the follow steps need to be specified on the file:

* The words and their respective rates need to be added to the [words/cuss](cuss) project, which will be used by the [get-alex/alex](alex) project to detect profane words;

* The insensitive phrases need to be added to the [retext-equality](equality) project, together with it suggested alternatives.

* Some important things to keep in mind:

   * On [retext-equality](equality) project, the phrases are added by hand;

   * Only phrases in [retext-equality](equality) have alternative suggestions.

[contributing]: https://github.com/get-alex/alex/blob/master/contributing.md

[cuss]: https://github.com/words/cuss

[alex]: https://github.com/get-alex/alex

[equality]: https://github.com/retextjs/retext-equality

**Comment by OtacilioN** [comment]
Hey @GledsonAfonso are you already writing this information in the guide? Or are you opening this issue to the community make it?

**Comment by GledsonAfonso** [comment]
It's open for the community, @OtacilioN, you can take it if you want. I opened the issue because I probably will not have the time to do it this weekend.

**Comment by Murderlon** [comment]
Agreed. Thanks to Hacktoberfest there have been lots of internationalization additions and discussions and this could help to keep that up 👍
